src/helpers/CommonCLI.h

```diff
@@ -48,6 +48,7 @@ struct NodePrefs { // persisted to file
   uint8_t advert_loc_policy;
   uint32_t discovery_mod_timestamp;
   float adc_multiplier;
+  uint32_t idle_interval; // in seconds
```

Comment:
Could we make this `idle_interval_seconds` and not need the comment?

Reply:
We could and I thought about it. In the end I decided to follow the style of the existing variables.

Reply:
I will leave that as is for now but happy to change it if others are in favor, too.